pkg/kubelet/kubelet.go

```diff
@@ -350,6 +352,31 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	if err != nil {
 		return nil, err
 	}
+	if len(thresholds) > 0 {
```

Comment:
I could see there or some NodeConfig type object that holds if swap is enabled that we can pass into EvictionManager and then have it error there...